holoviews/element/graphs.py

```diff
@@ -36,22 +42,37 @@ class layout_nodes(Operation):
     """
     Accepts a Graph and lays out the corresponding nodes with the
     supplied networkx layout function. If no layout function is
-    supplied uses a simple circular_layout function.
+    supplied uses a simple circular_layout function. Also supports
+    LayoutAlgorithm function provided in datashader layouts.
     """
 
+    as_graph = param.Boolean(default=False, doc="""
+        Whether to return Nodes or Graph.""")
```

Comment:
I don't particularly like this name. Why not simply ``graph``?

Reply:
I don't see this parameter used in this PR. Where are you expecting it to be used?

Reply:
Sure, I'll rename. It's used in ``if self.p.as_graph:`` below.

Reply:
I know it is used within this ``redim`` utility. What I'm wondering is where is redim used with this parameter set to ``True`` (assuming it is used with the default value by default).

Reply:
It's not it's used in the ``layout_nodes`` operation, this PR doesn't touch ``redim`` at all.

Reply:
Anyway how about ``only_nodes`` or ``nodes_only`` instead of ``as_graph``?

Reply:
Ah, I got confused and thought it was in ``redim`` for some reason so you can ignore that comment. I prefer ``only_nodes`` though as nodes are always returned how about ``with_edges``?

Reply:
I've gone with ``only_nodes`` and defaulted to False.